app/assets/stylesheets/custom.css.scss

```diff
@@ -181,3 +181,31 @@ input {
     }
   }
 }
+
+/* microposts */
+
+.microposts {
+  list-style: none;
+  margin: 10px 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `10px 0 0` instead of `10px 0 0 0`